### DISCUSSION TITLE
[Feature] <对外部库的照片进行操作>

### DISCUSSION THREAD

**Discussion by AlleNPooh** [immich-app/immich#6139] [Feature Request]
### The feature

由于之前使用的是其他的照片管理程序Synology photos，转到immich来的，在很多方面immich的优势明显，但是对外部库存在的大量照片没有办法导入到内部库，也没有办法进行操作。希望对这个功能进行加强。

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Please keep discussions in English, none of the maintainers speak Chinese.
